### DISCUSSION TITLE
Error in prisma deploy, first time user

### DISCUSSION THREAD

**Discussion by jdevng** [prisma/prisma#3468] [Q&A] [answered]
The error below, MongoSocketOpenException, goes back a few years, but the usual methods of ssl, localhost vs ip, removing schema do not seem to allow the connection despite mongo compass connecting fine. Looks like docker is not able to find the mongodb server, which is why prisma deploy eventually fails.

Also, the slack channel introduction link to github results in a 404.

Appreciate any feedback.

version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.34
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        # uncomment the next line and provide the env var PRISMA_MANAGEMENT_API_SECRET=my-secret to activate cluster security
        # managementApiSecret: my-secret
        databases:
          default:
            connector: mongo
            database: db
            ssl: true
            uri: >-
              mongodb://127.0.0.1:27017/db

>db_prisma % docker-compose up 
Starting db_prisma_prisma_1 ... done
Attaching to db_prisma_prisma_1
prisma_1  | No log level set, defaulting to INFO.
prisma_1  | [INFO] Cluster created with settings {hosts=[127.0.0.1:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500}
prisma_1  | [INFO] Exception in monitor thread while connecting to server 127.0.0.1:27017
prisma_1  | Exception opening socket
prisma_1  | com.mongodb.MongoSocketOpenException: Exception opening socket
prisma_1  | [INFO] Initializing workers...
prisma_1  | [INFO] Obtaining exclusive agent lock...
prisma_1  | [INFO] Obtaining exclusive agent lock... Successful.
prisma_1  | [INFO] Successfully started 1 workers.
prisma_1  | [INFO] No server chosen by com.mongodb.async.client.ClientSessionHelper$1@31337ee6 from cluster description ClusterDescription{type=UNKNOWN, connectionMode=SINGLE, serverDescriptions=[ServerDescription{address=127.0.0.1:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.ConnectException: Connection refused}}]}. Waiting for 30000 ms before timing out
prisma_1  | Server running on :4466

**Comment by ryands17** [comment, marked as answer]
Hey @jdevng 👋 
Could you add your issue [here](https://github.com/prisma/prisma1/issues) as this discussion is specific to Prisma 2.

**Reply by jdevng** [reply]
will do. as i mentioned the prisma introduction link in slack is 404. Thank you for responding.

**Reply by jdevng** [reply]
Actually that link goes to prisma/prisma1.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
